### ISSUE TITLE
730 Mode: 15% or 18% Bandwidth

### ISSUE THREAD

**Issue by kian1377** [uasal/webbpsf-stsci#3]
The phaseb data README file says the 730 mode is designed for 18% bandwidth but the table displaying 3 modes of operation says it should be 15%. We should find out which is correct.

**Comment by Jashcraf** [comment]
Good catch - I'm inclined to side with the phaseb data distribution, but lets find out

**Comment by Jashcraf** [comment]
@kian1393 https://roman.gsfc.nasa.gov/observatory.html information on the bandpasses from NASA, uses the following:

HLC mode -> 546-604nm
SPC mode 1 -> 675-785nm
SPC mode 2 -> 784-866nm
